build.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+# . "$here\$sut"
+$script = "$here\$sut"
+
+& $script -ErrorAction Stop -PowerShellModuleName PowerShellGet -PowerShellModuleVersion "2.1.3"
+
+Describe $script {
+  It "creates a module" {
```

Comment:
For any of these, do we want to verify any contents or just existance?

Reply:
validation of the files is done with `pdk validate` later. Not sure how brittle-vs-useful specific content checks would be.